src/pages/DashboardPage.js

```diff
@@ -131,30 +143,39 @@ const DashboardPage = () => {
                 <PanelLoaderWrapper isLoading={isVolumeAndTvlLoading}>
                   <StackedChart title={'Volume'} type={'BAR'} data={formattedVolumeData} />
                 </PanelLoaderWrapper>
-                <PanelLoaderWrapper isLoading={isComulativeDataLoading}>
+                <CardLoaderWrapper isLoading={isComulativeDataLoading}>
                   <ComulativeNetworkDataCard
                     title={'All time volume'}
                     icon={<DollarSign size={22} color={'#50dfb6'} />}
                     comulativeValue={`$ ${formattedNum(comulativeData.totalVolume)}`}
                     networksValues={formattedComulativeData.volume}
                   />
-                </PanelLoaderWrapper>
-                <PanelLoaderWrapper isLoading={isComulativeDataLoading}>
+                </CardLoaderWrapper>
+                <CardLoaderWrapper isLoading={isComulativeDataLoading}>
                   <ComulativeNetworkDataCard
                     title={'Total transactions'}
                     icon={<FileText size={22} color={'#50dfb6'} />}
                     comulativeValue={formattedNum(comulativeData.totalTrades)}
                     networksValues={formattedComulativeData.trades}
                   />
-                </PanelLoaderWrapper>
-                <PanelLoaderWrapper isLoading={isLoadingOneDayTransactions}>
+                </CardLoaderWrapper>
+                <CardLoaderWrapper isLoading={isLoadingOneDayTransactions}>
                   <NetworkDataCardWithDialog
                     title={'Trades (past 24h)'}
                     icon={<FileText size={22} color={'#50dfb6'} />}
                     dialogContent={'content'}
                     networksValues={oneDayTransactions}
                   />
-                </PanelLoaderWrapper>
+                </CardLoaderWrapper>
+                <CardLoaderWrapper isLoading={isLoadingOneDayWallets}>
+                  <NetworkDataCardWithDialog
+                    title={'Wallets (past 24h)'}
+                    chartTitle={'Wallets'}
+                    icon={<Users size={22} color={'#50dfb6'} />}
```

Comment:
One suggestion is to move these icons to a separate file or (put each icon in icons folder and reexport the theme-wrapped icons) wrap it in styled and use color from theme. So they will be theme aware later if we need to switch to light/dark mode